src/main/java/uk/gov/hmcts/divorce/caseworker/service/print/ApplicationPrinter.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.divorce.caseworker.service.print;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.document.print.BulkPrintService;
+import uk.gov.hmcts.divorce.document.print.model.Letter;
+import uk.gov.hmcts.divorce.document.print.model.Print;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+import static org.springframework.util.CollectionUtils.firstElement;
+import static uk.gov.hmcts.divorce.document.DocumentUtil.lettersWithDocumentType;
+import static uk.gov.hmcts.divorce.document.model.DocumentType.APPLICATION;
+
+@Component
+@Slf4j
+public class ApplicationPrinter {
+
+    private static final String DIVORCE_APPLICATION_PDF = "divorce-application-pdf";
```

Comment:
I'm sure I'm splitting hairs but shouldn't this just be application-pdf because it might be a dissolution

Reply:
good point!